core/src/consensus/tendermint/worker.rs

```diff
@@ -1835,11 +1868,45 @@ impl Worker {
                 _ => {}
             }
 
+            // priority verification block
+            {
+                let parent_seed = self.prev_vrf_seed_of_height(number)?;
+                let signer_idx = priority_message.seed_signer_idx();
+
+                match priority_message.verify_seed(
+                    number,
+                    proposed_view,
+                    &parent_seed,
+                    &signer_public,
+                    &mut self.sortition_scheme.vrf_inst,
+                ) {
+                    Ok(true) => {}
+                    _ => {
+                        cwarn!(ENGINE, "Proposal VRF seed verification failed");
+                        return None
+                    }
+                };
```

Comment:
Let's use `Result::is_err(&self)`.
```
if priority_message.verify_seed(
    number,
    proposed_view,
    &parent_seed,
    &signer_public,
    &mut self.sortition_scheme.vrf_inst,
).is_err() {
        cwarn!(ENGINE, "Proposal VRF seed verification failed");
        return None
};
```

Reply:
I just want to aggregate both cases `Ok(false)` and `Err(..)`